src/plugins/intel_cpu/src/nodes/pooling.cpp

```diff
@@ -373,7 +373,7 @@ void Pooling::prepareParams() {
                 break;
             }
             if (!itpd.next_impl())
-                return nullptr;
+                break;
```

Comment:
I'm not sure, but logically, when the iterator is over, the while condition should become false, so this check probably does not make sense.

Reply:
I looked through the iterator behavior and can conclude that we need this check because `static_cast<bool>(itpd)` never returns `false`. Also, I checked all places where we have the same pd traversal logic: we use such loop interruption condition everywhere. So suggest leaving as is

Reply:
Got it, thank you.